com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/PathTracing/PathTracing.cs

```diff
@@ -571,6 +698,8 @@ TextureHandle RenderPathTracing(RenderGraph renderGraph, HDCamera hdCamera, Text
             }
 #endif
 
+            pathTracedAOVs.Clear();
```

Comment:
Couldn't this be moved at the end of the previous ENABLE_UNITY_DENOISING_PLUGIN block?